src/main/common/filter.c

```diff
@@ -117,7 +77,67 @@ float rateLimitFilterApply4(rateLimitFilter_t *filter, float input, float rate_l
     return filter->state;
 }
 
-// FIR filter
+float filterGetNotchQ(uint16_t centerFreq, uint16_t cutoff)
```

Comment:
Me too. Theres so much in there and I feel I probably only know about half of the changes that have happened. But @borisbstyle has said the the notch filter was one thing that surprised him and gave really good improvements in flight characteristics, so it's probably something we should add here soon. The 2DOF stuff I haven've even looked at yet. And perhaps @joshuabardwell will do a video about some of the changes I have made, so that I will be able to understand them :-)